packages/web-app-files/src/components/SideBar/TagsPanel.vue

```diff
@@ -74,6 +76,7 @@ type TagOption = {
 
 export default defineComponent({
   name: 'TagsPanel',
+  methods: { keycode },
```

Comment:
Please don't introduce new options api code. You can return the keycode function in the setup function.

Reply:
hmm, I should better not rely on method importing using WebStorm or at least I should double check before committing :/